tests/test_aio.py

```diff
@@ -12,21 +13,23 @@
 
 @pytest.fixture()
 async def consul_obj(consul_port):
+    consul_port, consul_version = consul_port
```

Comment:
Confusing way of unpacking - one wouldn't expect the version in port - why not add the port as add arg?